### PR TITLE
Resolved all of the listed issues

### PR DESCRIPTION
**Pull request by Litery**

### CODE REVIEW

File: luncher/meals/views.py [hunk 1]
```diff
@@ -1,16 +1,57 @@
...
+        self.to_json = to_json or jsonify
 
     def get(self):
-        data, _ = self.serializer.dump(self.dao.all())
-        return jsonify(data), HTTPStatus.HTTP_200_OK
+        data, _ = self.meals_schema.dump(self.dao.join(Meal.venue))
+        return self.to_json(data), HTTPStatus.HTTP_200_OK
+
+    @json_response
+    def post(self):
```
Comment (piotrek-py) [inline luncher/meals/views.py]: How about:

> You should create decorator `@authorization_required` to use in controller methods

from  [issue #5](https://github.com/rstit/luncher/issues/5)

File: luncher/meals/views.py [hunk 2]
```diff
@@ -1,16 +1,57 @@
...
+        token = request.headers.get('Authorization')
+        if not token:
+            return self._unauthorized
+
+        user_id = self.token_helper.read_user_id(token)
+        if not user_id or self.dao.get_by_ids([user_id]).count() == 1:
```
Comment (piotrek-py) [inline luncher/meals/views.py]:
```python
self.dao.get_by_ids([user_id]).count() == 1
```
1. Are you sure you are using valid Dao?
2. You should create a method with business logic in the user's dao.
3. Are you sure this is good `self.dao.get_by_ids([user_id]).count() == 1`. Do you have any tests that test this?

File: luncher/meals/views.py [hunk 3]
```diff
@@ -1,16 +1,57 @@
...
+                self.dao.save(validated_entity)
+                return 'ok', HTTPStatus.HTTP_201_CREATED
+
+        return self._make_error('Bad request', HTTPStatus.HTTP_400_BAD_REQUEST)
+
+    @property
+    def _unauthorized(self):
+        return self._make_error('Unauthorized', HTTPStatus.HTTP_401_UNAUTHORIZED)
+
+    @staticmethod
```
Comment (piotrek-py) [inline luncher/meals/views.py]: Why do you need `staticmethod` here?

Comment (Litery) [inline luncher/meals/views.py]: It doesn't need the context of an object.

**Pull request closed without merging**